packages/azure-container-services/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@instana/azure-container-services",
+  "version": "2.36.1",
+  "description": "Instana tracing and monitoring for Node.js based Azure Container Services",
+  "author": {
+    "name": "Arya Mohanan",
+    "email": "arya.mohanan@ibm.com"
+  },
+  "homepage": "https://github.com/instana/nodejs/blob/main/packages/azure-container-services/README.md",
+  "main": "src/index.js",
+  "files": [
+    "src",
+    "esm-loader.mjs"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "directories": {
+    "lib": "src",
+    "test": "test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/instana/nodejs.git"
+  },
+  "scripts": {
+    "audit": "npm audit --omit=dev",
+    "test": "NODE_ENV=debug mocha --sort $(find test -iname '*test.js' -not -path '*node_modules*')",
+    "test:debug": "WITH_STDOUT=true npm run test",
+    "test:ci": "echo \"******* Files to be tested:\n $CI_AZURE_CONTAINER_SERVICES_TEST_FILES\" && if [ -z \"${CI_AZURE_CONTAINER_SERVICES_TEST_FILES}\" ]; then echo \"No test files have been assigned to this CircleCI executor.\"; else mocha --reporter mocha-multi-reporters --reporter-options configFile=reporter-config.json --sort ${CI_AZURE_CONTAINER_SERVICES_TEST_FILES}; fi",
+    "lint": "eslint src test images",
+    "verify": "npm run lint && npm test",
+    "prettier": "prettier --write 'src/**/*.js' 'test/**/*.js'"
+  },
+  "contributors": [
+    {
+      "name": "Arya Mohanan",
+      "email": "arya.mohanan@ibm.com"
+    },
+    {
+      "name": "Katharina Irrgang",
+      "email": "katharina.irrgang@ibm.com"
+    }
+  ],
+  "bugs": {
+    "url": "https://github.com/instana/nodejs/issues"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@instana/core": "2.36.1",
+    "@instana/serverless": "2.36.1"
```

Comment:
TODO: I think we also need to manually update these versions after the rebase

Reply:
done